engine/src/stack.h

```diff
@@ -1083,7 +1084,9 @@ class MCStack : public MCObject
 	void SetSystemWindow(MCExecContext& ctxt, bool setting);
 	void GetMetal(MCExecContext& ctxt, bool& r_setting);
 	void SetMetal(MCExecContext& ctxt, bool setting);
+	using MCObject::GetShadow;
```

Comment:
Without the `using` declaration, the virtual method `MCObject::SetShadow(MCExecContext &, const MCInterfaceShadow &)` (fbase1) is masked by the overloaded virtual method `MCStack::SetShadow(MCExecContext &, bool)` (fderived2). In order to avoid ambiguity, either fbase needs to be used in the derived class (to create a vtable entry), or overridden with some implementation fderived1, or fderived2 needs to be renamed to remove the overload.  I went with option 1 because it makes the current behaviour explicit.

As a third option, I could rename `MCStack::SetShadow(MCExecContext &, bool)` to `MCStack::SetWindowShadow`.